forward/private/leadfield_duneuro.m

```diff
@@ -0,0 +1,51 @@
+function [lf] = leadfield_duneuro(pos, vol, method)
```

Comment:
is this function not beter located in `external/duneuro`?

Reply:
I guess that leadfield_duneuro.m was put in `forward/private` because all other functions leadfield_{fns, interpolate, simbio}.m are there. Shall I move it anyways?

Reply:
no, you can leave it in private